packages/headless/src/features/generated-answer/generated-answer-slice.ts

```diff
@@ -68,12 +75,19 @@ export const generatedAnswerReducer = createReducer(
         state.feedbackSubmitted = true;
       })
       .addCase(resetAnswer, (state) => {
-        return {
-          ...getGeneratedAnswerInitialState(),
-          responseFormat: state.responseFormat,
-          fieldsToIncludeInCitations: state.fieldsToIncludeInCitations,
-          isVisible: state.isVisible,
-        };
+        return handleResetAnswer(state);
+      })
+      .addCase(executeSearch.fulfilled, (state, payload) => {
```

Comment:
@nathanlb I remember you told me Friday about a situation where triggering live streams on   `executeSearch.fulfilled` was not enough, can you remember why?

Reply:
I remember considering it but unfortunately I can't quite remember if there was a specific situation this omitted. It's possible that I had moved on because it still required the abortController to be handled somewhere else and I just didn't get back to it and consider this particular setup involving "should" flags.
We should do some QA testing but in theory I think this works.